application/apache-kafka-flink-streaming/flink_consumer.py

```diff
@@ -0,0 +1,67 @@
+import os
+import logging
+
+from pathlib import Path
+
+from pyflink.common import Types
+from pyflink.datastream import StreamExecutionEnvironment
+from pyflink.datastream.connectors.jdbc import JdbcSink, JdbcConnectionOptions, JdbcExecutionOptions
+from pyflink.datastream.connectors.kafka import FlinkKafkaConsumer
+from pyflink.datastream.formats.json import JsonRowDeserializationSchema
+
+logging.basicConfig(
+    format='[%(asctime)s] [%(levelname)s] [%(filename)s:%(lineno)d] - %(message)s',
+    level=logging.DEBUG
+)
+
+JARS_PATH = Path(__file__).parent / 'jars'
+
+KAFKA_BOOTSTRAP_SERVER = os.getenv('FLINK_CONSUMER_BOOTSTRAP_SERVER')
+KAFKA_TOPIC = os.getenv('FLINK_CONSUMER_KAFKA_TOPIC')
+CRATEDB_PG_URI = os.getenv('FLINK_CONSUMER_CRATEDB_PG_URI', 'jdbc:postgresql://localhost:5432/crate')
+CRATEDB_USER = os.getenv('FLINK_CONSUMER_CRATE_USER')
+CRATEDB_PASSWORD = os.getenv('FLINK_CONSUMER_CRATE_PASSWORD')
```

Comment:
Just another nit: Let's use a fully-qualified `CRATEDB` everywhere?
```suggestion
CRATEDB_USER = os.getenv('FLINK_CONSUMER_CRATEDB_USER')
CRATEDB_PASSWORD = os.getenv('FLINK_CONSUMER_CRATEDB_PASSWORD')
```